install_workflow/create_env_nurion.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
Is it possible to generate pip packages installed, and reverse-engineer the environment?